ts/util/html-utils.ts

```diff
@@ -0,0 +1,36 @@
+// adds a header cell to a table header
+export function addColumnHeader(
+  thead: HTMLTableRowElement,
+  header: string,
+  index: number
+) {
+  const th = document.createElement("th");
+  th.className = "p-4 text-white font-semibold bg-grey-300";
+  if (index > 0) {
+    th.classList.add("border-grey-200", "border-l-2");
+  }
+  th.appendChild(document.createTextNode(header));
+  thead.appendChild(th);
+}
+
+// adds link-styled text to a div
+export function addLink(container: HTMLElement, text: string, url: string) {
+  const a = document.createElement("a");
+  a.setAttribute("href", url);
+  a.className = "text-green-200 font-bold hover:underline";
+  a.innerHTML = text;
+  container.appendChild(a);
+}
+
+// updates an empty col with corresponding style
+export function styleEmptyColumn(td: HTMLTableCellElement) {
+  td.className = "text-grey-300 bg-grey-200 border-grey-200 border-t-2";
+  td.appendChild(document.createTextNode("N/A"));
+}
```

Comment:
This function is doing a lot more than it should

* Setting the contents ("N/A") will normally be handled by the getParent/ChildContents functions. It's only for cases where a table has child columns and a parent object has no children that the table builder should set "N/A" itself. Move adding the text node back to the table builder
* "styleEmptyColumn" is unclear - "N/A" is "empty"? It's also only styling a cell, not an entire column. Rename to "styleNotApplicableCell"
* top cell borders (`border-grey-200 border-t-2`) are added in 4 different places. Remove them from here and instead, add a "addCell(tr)" function to table builder that calls tr.insertCell, sets these 2 classes, and returns the cell. Then change table-builder to call that function wherever it's currently calling tr.insertCell. `p-4` should probably be included for all cells too